app/features/settings/settings.tsx

```diff
@@ -8,23 +13,77 @@ import {
 import { Button } from '~/components/ui/button';
 import { Separator } from '~/components/ui/separator';
 import { SettingsNavButton } from '~/features/settings/ui/settings-nav-button';
+import useLocationData from '~/hooks/use-location';
+import { useToast } from '~/hooks/use-toast';
 import { canShare, shareContent } from '~/lib/share';
-import { LinkWithViewTransition } from '~/lib/transitions';
+import { cn } from '~/lib/utils';
 import { useDefaultAccount } from '../accounts/account-hooks';
 import { AccountTypeIcon } from '../accounts/account-icons';
 import { ColorModeToggle } from '../theme/color-mode-toggle';
 import { useAuthActions } from '../user/auth';
 import { useUser } from '../user/user-hooks';
 
+function UsernameWithCopyableLightningAddress({
```

Comment:
nit but why is it called Username if it displays and copies lightning address? I'd maybe just call it LnAddressDisplay or CopyableLnAddress